### PR TITLE
chore: update luadoc parser & highlights

### PR DESCRIPTION
**Pull request by amaanq**

### CODE REVIEW

File: queries/luadoc/highlights.scm
Comment (lucario387) [inline queries/luadoc/highlights.scm:1]: What about member_type :thinking: 

I have this in my highlight (java-like scoping)
```scheme
(member_type
  (identifier) @namespace)
(member_type
  (identifier) @type .)
```

Comment (amaanq) [reply queries/luadoc/highlights.scm:1]: yeah that could be a nice addition

Comment (lucario387) [reply queries/luadoc/highlights.scm:1]: Oops
It should be module :smile:

File: queries/luadoc/highlights.scm [hunk 1]
```diff
@@ -93,6 +98,12 @@
   .
   (identifier) @variable.member)
 
+(member_type
+  (identifier) @namespace)
```
Comment (lucario387) [inline queries/luadoc/highlights.scm:102]:
```suggestion
  (identifier) @module)
```

**Pull request merged**